wrangler-core/src/main/java/io/cdap/directives/external/InvokeHttp.java

```diff
@@ -96,21 +96,18 @@ public void initialize(Arguments args) throws DirectiveParseException {
         String[] components = header.split("=");
         if (components.length != 2) {
           throw new DirectiveParseException (
-            String.format("Incorrect header '%s' specified. " +
-                            "Header should be specified as 'key=value' pairs separated by a comma (,).", header)
-          );
+            NAME, String.format("Incorrect header '%s' specified. Header should be specified as 'key=value' " +
```

Comment:
Can we enhance the message to also include the column name?

Same with the other messages in this class

Reply:
Did not add column name because this directive expects list of columns.